### ISSUE TITLE
NagadException: Please verify provided data and try again. Call 16167 for help.

### ISSUE THREAD

**Issue by saaiful** [shahriar-shojib/nagad-payment-gateway#14]
Geting error: NagadException: Please verify provided data and try again. Call 16167 for help.
My Credential is Live
Using Base URL: https://api.mynagad.com

**Comment by saaiful** [comment]
Issue solved using: 
```
const month = now.getMonth() + 1 < 10 ? `0${now.getMonth() + 1}` : `${now.getMonth() + 1}`;
```

BTW, NagadException message should be `data.devMessage`

**Comment by shahriar-shojib** [comment]
fixed in [v2.0.2](https://github.com/shahriar-shojib/nagad-payment-gateway/releases/tag/v2.0.2)
